core/lib/spree/app_configuration.rb

```diff
@@ -389,6 +389,10 @@ def stock
       @stock_configuration ||= Spree::Core::StockConfiguration.new
     end
 
+    def event_bus
+      @event_bus_class ||= Spree::EventBus.instance
+    end
```

Comment:
I don't think this should live in `Spree::Config`. The class is not configurable, but the name implies it is.

Defining this in `Spree.event_bus` should be sufficient.